src/utils.ts

```diff
@@ -13,10 +13,12 @@ import {
   STATS_CATEGORIES, GITHUB_ASSETS_URL
 } from './constants';
 
-export const getAvatarURL = (id: UUID, size = 256) =>
-  `${AVATARS_URL}/${id}/default_${size === 500 ? 'tall' : `${size}_${size}`}.png`;
+export const getAvatarURL = (id: UUID | null, size = 256) =>
+  id
```

Comment:
This check is probably unnecessary since `https://ubisoft-avatars.akamaized.net/null/default_146_146.png` will still return desired "default" profile picture